components/director/pkg/graphql/schema.graphql

```diff
@@ -1738,7 +1738,7 @@ type Mutation {
 	writeTenant(in: BusinessTenantMappingInput!): String! @hasScopes(path: "graphql.mutation.writeTenants")
 	deleteTenants(in: [String!]): Int! @hasScopes(path: "graphql.mutation.deleteTenants")
 	updateTenant(id: ID!, in: BusinessTenantMappingInput!): Tenant! @hasScopes(path: "graphql.mutation.updateTenant")
-	subscribeTenant(providerID: String!, subaccountID: String!, providerSubaccountID: String!, consumerTenantID: String!, region: String!, subscriptionAppName: String!): Boolean! @hasScopes(path: "graphql.mutation.subscribeTenant")
+	subscribeTenant(providerID: String!, subaccountID: String!, providerSubaccountID: String!, consumerTenantID: String!, region: String!, subscriptionAppName: String!, subscriptionPayload: String): Boolean! @hasScopes(path: "graphql.mutation.subscribeTenant")
```

Comment:
This is very minor - maybe it's a good idea to make the `subscriptionPayload` argument mandatory? (by using `String!`)

As far as I understand the only caller of this `subscribeTenant` mutation is the tenant fetcher in the context of SaaS  onSubscribe callbacks - these callbacks will always have a payload and in the code of the tenant fetcher if made it so that it is always sent along this mutation (as it should be). In this sense this property will always be provided?

Reply:
My only concern is someone in the future looking at this code and wondering why is the `subscriptionPayload` parameter not mandatory and wondering if there is some other use case where it is not provided :)